source/guide_seafile.rst

```diff
@@ -101,9 +103,8 @@ Run the seafile installer script.
 
 ::
 
- [isabell@stardust ~]$ cd ~/seafile/seafile-server-*
- [isabell@stardust ~]$ ./setup-seafile-mysql.sh
- [isabell@stardust ~]$
+ [isabell@stardust seafile]$ ( cd ~/seafile/seafile-server-* && ./setup-seafile-mysql.sh )
```

Comment:
I'd like to keep the commands in two lines, makes it easier for novice users to understand what they're doing.